frontend/src/views/Landing/index.tsx

```diff
@@ -52,8 +52,8 @@ const LandingPage = (): JSX.Element => {
   const scrollSection5 = useRef<HTMLDivElement>(null!);
 
   const [cellsHeroNum] = useState("38.1M");
```

Comment:
just to make sure, the number of cells still rounds to 38.1M?

Reply:
yup total number of unique cells is 38,157,179

Reply:
51,645 added from last week